src/components/enterprise-invite/EnterpriseInvitePage.test.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
-import { getConfig } from '@edx/frontend-platform/config';
 import { screen, waitFor } from '@testing-library/react';
+import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
```

Comment:
EnterpriseInvitePage is being wrapped by [LoginRedirect](https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/master/src/components/enterprise-invite/EnterpriseInvitePage.jsx#L75) component of frontend-enterprise, this component [uses](https://github.com/openedx/frontend-enterprise/blob/master/packages/logistration/src/LoginRedirect.jsx#L21) the `getAuthenticatedUser` which is why it is still being mocked over here.

This can also be handled by mocking `LoginRedirect` component. 

i.e.
`
jest.mock('@edx/frontend-enterprise-logistration', () => ({
  LoginRedirect: ({ children }) => children,
}));
`
What do you suggest? Should we mock `getAuthenticatedUser` or `LoginRedirect`?

Reply:
@Syed-Ali-Abbas-Zaidi Ah, thanks for clarifying why it's still mocked here. I'd say let's mock `LoginRedirect` at this point, given its from an external NPM package and should have its own [test suite(s)](https://github.com/openedx/frontend-enterprise/blob/master/packages/logistration/src/LoginRedirect.test.jsx).